src/Illuminate/Log/Context/Repository.php

```diff
@@ -360,7 +360,7 @@ public function hiddenStackContains(string $key, mixed $value, bool $strict = fa
         }
 
         if ($value instanceof Closure) {
-            return $value($this->data[$key]);
+            return collect($this->hidden[$key])->contains($value);
```

Comment:
This is a bugfix as well because it was not checking the hidden data.

Reply:
I was already going to notify you about this after I read the other PR 😂